catalogue/webapp/pages/api/works/items.ts

```diff
@@ -0,0 +1,47 @@
+import fetch from 'isomorphic-unfetch';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { ItemsWork, CatalogueApiError } from '@weco/common/model/catalogue';
+import { catalogueApiError } from '../../../services/catalogue/common';
+import hasOwnProperty from '@weco/common/utils/has-own-property';
+
+export function isCatalogueApiError(
+  response: ItemsWork | CatalogueApiError
+): response is CatalogueApiError {
+  return Boolean(hasOwnProperty(response, 'type') && response.type === 'Error');
+}
+
+async function fetchWorkItems(
+  id: string
+): Promise<ItemsWork | CatalogueApiError> {
+  try {
+    const items = await fetch(
+      `https://api.wellcomecollection.org/catalogue/v2/works/${id}/items`,
```

Comment:
This PR has already been heavily reviewed - but this does need to be toggleable to stage for the e2e tests to work effectively when deploying the catalogue api (but if there are not e2e tests for this yet, the point is moot).

Anyway - not blocking for this PR.